<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagesFrame</class>
 <widget class="QFrame" name="MessagesFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="m_messagesView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>0</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property> 
     <property name="styleSheet">
      <string notr="true">QTableView::item
{
  border: 0px;
  border-bottom: 1px solid #e1e1e1;
  padding: 10px 5px 10px 5px;
  spacing: 0px;
  margin: 0px;
}
QTableView::item:selected
 {
     background-color:#eee;
     color: black;
     border:none;
     border-bottom: 1px solid #e1e1e1;
     outline:none;
 }
 QTableView::item:focus
 {
    selection-background-color:transparent;
    selection-border:transparent;
    outline:none;
    border:none;
 }

QTableView::item:selected:active {
    background-color:purple;
    outline:none;
    border:none;
} 

QTableView {
    outline:none;
} 
</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_addContactButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>         
       <property name="text">
        <string> Add Contact</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/contact-new</normaloff>:/icons/contact-new</iconset>
       </property>       
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_detailsButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>         
       <property name="text">
        <string> View Message Details</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/details</normaloff>:/icons/details</iconset>
       </property>       
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_replyButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>         
       <property name="text">
        <string> Reply</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/reply</normaloff>:/icons/reply</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_messagesView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MessagesFrame</receiver>
   <slot>messageDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_replyButton</sender>
   <signal>clicked()</signal>
   <receiver>MessagesFrame</receiver>
   <slot>replyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_detailsButton</sender>
   <signal>clicked()</signal>
   <receiver>MessagesFrame</receiver>
   <slot>showDetailsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addContactButton</sender>
   <signal>clicked()</signal>
   <receiver>MessagesFrame</receiver>
   <slot>addContactClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>

 </connections>
 <slots>
  <slot>messageDoubleClicked(QModelIndex)</slot>
  <slot>replyClicked()</slot>
  <slot>showDetailsClicked()</slot>
  <slot>addContactClicked()</slot>
</slots>
</ui>
